src/main/java/org/generationcp/middleware/dao/UserDefinedFieldDAO.java

```diff
@@ -37,12 +38,12 @@
 	private static final Logger LOG = LoggerFactory.getLogger(UserDefinedFieldDAO.class);
 
 	@SuppressWarnings("unchecked")
-	public List<UserDefinedField> getByFieldTableNameAndType(final String tableName, final String fieldType) {
+	public List<UserDefinedField> getByFieldTableNameAndType(final String tableName, final Set<String> fieldType) {
```

Comment:
Even when we may end not using this function I am in favor of using Set instead of String :+1: